keyboards/index.php

```diff
@@ -46,14 +56,14 @@
 
 <div class='<?= $embed == 'none' ? '' : 'embed embed-'.$embed ?>'>
 
-  <h2 class="red underline"><a href='/keyboards'>Keyboard Search</a></h2>
+  <h2 class="red underline"><a href='/keyboards'><?= _("Keyboard Search") ?></a></h2>
 
   <div id='search-box'>
     <form method='get' action='/keyboards' name='f'>
-      <label for="search-q">Keyboard search:</label><input id="search-q" type="text" placeholder="Enter language or keyboard" name="q"
+      <label for="search-q"><?= _s("Keyboard search%s", ":") ?></label><input id="search-q" type="text" placeholder="<?= _("Enter language or keyboard")?>" name="q"
       <?php if($embed == 'none') echo 'autofocus'; ?>>
-      <input id="search-f" type="image" src="<?= cdn('img/search-button.png"') ?>" value="Search" onclick="return do_search()">
-      <label id="search-new"><a href='/keyboards<?=$session_query_q?>'>New search</a></label>
+      <input id="search-f" type="image" src="<?= cdn('img/search-button.png"') ?>" value="<?= _("Search") ?>" onclick="return do_search()">
```

Comment:
The screenshots show the search button still in English - 
It's because we're using a static image
https://github.com/keymanapp/keyman.com/blob/master/cdn/dev/img/search-button.png

Reply:
I could convert to a label, if someone can provide some CSS-fu styling

```html
<label id="search-f" class="keyboard_search_button" alt="search button" onclick="return do_search()"> <?= _("Search") ?> </label>
```